### PR TITLE
Tailwind CSS integration

### PR DESCRIPTION
**Pull request by aluarius**
Closes: #904 

## Description
- [x] Vuestic UI version bump to 1.6.1.
- [x] Removed Vuestic UI grid CSS from imports.
- [x] Added Tailwind CSS.
- [x] All Vuestic UI helpers moved to Tailwind CSS.  
- [x] Related refactor and minor bugfixing. 

No need to inspect code - only QA.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,20 +153,57 @@ contribution guides</a>
 and [discord server](https://discord.gg/jTKTjj2weV)
 
 ### Partners & Sponsors ❤️
+
 <div>
-        <a href="https://epicmax.co/" target="_blank" title="Epicmax - Top Vue.js Development Company"><img src="./.github/assets/epicmax.svg" loading="lazy" vspace="10" alt=""></a>
-        <a href="https://www.ag-grid.com/" target="_blank" title=" Data Grid: AG Grid: High-Performance React Grid, Angular Grid, JavaScript Grid"><img hspace="15" vspace="10" src="./.github/assets/aggrid.svg" loading="lazy" alt=""></a>
-        <a href="https://vuejobs.com/" target="_blank" title="Vue.js jobs – Browse through dozens of Vue.js openings"><img src="./.github/assets/vuejobs.svg" loading="lazy" vspace="10" alt=""></a>
-        <a href="https://flatlogic.com/" target="_blank" title="Flatlogic - The best way to create React, Angular and Vue full-stack web applications"><img hspace="15" vspace="10" src="./.github/assets/flatlogic.svg" loading="lazy" alt=""></a>
+  <a href="https://vuejs.org/partners/epicmax.html" target="_blank" title="VueJS - The Progressive JavaScript Framework">
```
Comment (m0ksem) [inline README.md:158]: Junk?

**Review by m0ksem — CHANGES_REQUESTED** [review]

**Comment by aluarius** [comment]
> Lost some icons

Icon's color. It was before - didn't mention. Fixed.

About second screenshot - can't repro via two browsers and deps reinstallation. Please, check if it wasn't accidentally for you.

image

**Comment by m0ksem** [comment]
@aluarius, try firefox

**Comment by aluarius** [comment]
@m0ksem ah, i see, fixed.

**Review by m0ksem — APPROVED** [review]

**Pull request merged**